getting-started/docker-compose-trino.yml

```
@@ -0,0 +1,59 @@
+#
+# Copyright (c) 2024 Snowflake Computing Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+services:
+  polaris:
+    build:
+      context: ..
+    ports:
+      - "8181:8181"
+      - "8182"
+    environment:
+      AWS_REGION: us-west-2
+      GOOGLE_APPLICATION_CREDENTIALS: $GOOGLE_APPLICATION_CREDENTIALS
+      AZURE_TENANT_ID: $AZURE_TENANT_ID
+      AZURE_CLIENT_ID: $AZURE_CLIENT_ID
+      AZURE_CLIENT_SECRET: $AZURE_CLIENT_SECRET
+    command: # override the command to specify aws keys as dropwizard config
+      - java
+      - -Ddw.awsAccessKey=$AWS_ACCESS_KEY_ID
+      - -Ddw.awsSecretKey=$AWS_SECRET_ACCESS_KEY
+      - -jar
+      - /app/polaris-service-1.0.0-all.jar
+      - server
+      - polaris-server.yml
+
+    healthcheck:
+      test: ["CMD", "curl", "http://localhost:8182/healthcheck"]
+      interval: 10s
+      timeout: 10s
+      retries: 5
+
+  create-catalog:
+    image: curlimages/curl
+    depends_on:
+      polaris:
+        condition: service_healthy
+    volumes:
+      - ./create-catalog.sh:/create-catalog.sh
+    command: ["/bin/sh", "/create-catalog.sh"]
+
+  trino:
+    image: trinodb/trino:latest
+    ports:
+      - "8080:8080"
+    volumes:
+      - ./trino-config:/etc/trino
```

Why do we want to deploy non-catalog files (config.properties, jvm.config, node.properties)? I would change to like below and remove unnecessary files if there's no special reason:
```suggestion
      - ./trino-config/catalog:/etc/trino/catalog
```

---

thanks for the pointer, looks like the trino docker image has single node config by default. nice! I'll remove all the unnecessary configs